### PR TITLE
Added Swiftify Xcode Extension to the new "Extensions" category (separated from "Plugins" for clarity)

### PR DESCRIPTION
**Pull request by alex-swiftify**
## Why it should be included to `awesome-ios` (optional)
While the Swiftify Xcode Extension is currently closed source, it has thousands of users and it's a tool created primarily for iOS developers. 
Please accept a link to the AppStore instead of GitHub due to the above reason.
The documentation is available at https://objectivec2swift.com/#/home/xcode-extension/ and linked from the AppStore page.

## Checklist

- [ x] Only one project/change is in this pull request
- [ x] Addition in chronological order (bottom of category)
- [ x] Supports iOS 9 / tvOS 10 or later
- [ x] Supports Swift 3 or later
- [ x] Has a commit from less than 2 years ago
- [ x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 2 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
632 | [301](https://httpstatuses.com/301) | http://www.placekit.io/ redirects to<br>https://business.mapquest.com/products/points-of-interest-data/
2654 | [301](https://httpstatuses.com/301) | http://codewithchris.com/ redirects to<br>https://codewithchris.com/


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by lfarah** [comment]
Hey @alex-swiftify, thanks for that! Your PR definitely makes sense, but since it's a change bigger than adding a lib, I'd like to hear from @dkhamsing, @vsouza, @victorgama @miguelarauj1o  or @naeemshaikh90 before merging your PR

File: README.md [hunk 1]
```diff
@@ -147,6 +147,7 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
     - [Official](#official)
     - [Unofficial](#unofficial)
 - [Xcode](#xcode)
+    - [Extensions (Xcode 8+)](#extensions)
```
Comment (dkhamsing) [inline README.md]: `#extensions` would need to be changed to `#extensions-xcode-8`

Comment (alex-swiftify) [reply README.md]: Thanks! Updated to #extensions-xcode-8 as you requested.

**Comment by dkhamsing** [comment]
Thank u!

**Comment by lfarah** [comment]
Thanks guys! Great team work

**Pull request merged**